functions/New-DbaEndpoint.ps1

```diff
@@ -153,22 +160,18 @@ function New-DbaEndpoint {
                 }
             }
 
-            if ($Pscmdlet.ShouldProcess($server.Name, "Creating endpoint $Name of type $Type using protocol $Protocol and if TCP then using Port $tcpPort")) {
+            if ($Pscmdlet.ShouldProcess($server.Name, "Creating endpoint $Name of type $Type using protocol $Protocol and if TCP then using IPAddress $IPAddress and Port $tcpPort")) {
```

Comment:
I would think this a bit confusing to some users seeing the IP in the WhatIf output if it was not provided:

![image](https://user-images.githubusercontent.com/11204251/92663101-f3bdc200-f2c5-11ea-9676-e686f4d84804.png)

Some of the WhatIf messages are not ideal in our commands, so you are welcome to update/rearrange this to be more informative if you want. I would not see adding "Creating" until the actual call to the if block for certificate and `.Create()` call. We could just have this split to say "Setting..." for setting protocol, owner, IP (if provided), etc. Then when it goes to actual create have the "Creating...". I'd also wrap the call to `Get-DbaEndpoint` if we chose to rearrange so it does not execute that command.

Reply:
But '0.0.0.0' is the equivalent for the default "ALL" and that's what is there inside the smo if you don't set it - try with the old version of the module. That's why I don't have a `if ($IPAddress) { ... }` in there but just set the default of '0.0.0.0'.
If you would like to rephrase the -WhatIf messages: No problem, just please open your own pull request for that.